kafka-streams-quickstart/aggregator/src/main/java/org/acme/quarkus/sample/kafkastreams/streams/JsonbSerde.java

```diff
@@ -0,0 +1,103 @@
+package org.acme.quarkus.sample.kafkastreams.streams;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serializer;
+
+/**
+ * A {@link Serde} that (de-)serializes JSON.
+ */
+public class JsonbSerde<T> implements Serde<T> {
```

Comment:
I would consider this as a good addition to the Kafka client extension. WDYT?

Reply:
Yes, good idea. I already have filed https://github.com/quarkusio/quarkus/issues/2863 as a follow-up for the Kafka Streams extension. Just added this as one item there.